src/aap_eda/settings/default.py

```diff
@@ -204,6 +205,10 @@ def _get_boolean(name: str, default=False) -> bool:
 # See https://cfpb.github.io/django-flags/
 FLAGS = {}
 
+for feature in list(FLAGS.keys()):
```

Comment:
We don't need to call `list` here, right?

Reply:
You're right, I can remove list here, will update